openwisp_controller/connection/static/connection/js/commands.js

```diff
@@ -571,3 +575,7 @@ function getFormattedDateTimeString(DateTimeString) {
     stringArray[4] = (stringArray[4] == 'AM') ? 'a.m.' : 'p.m.';
     return stringArray.join(' ');
 }
+
+function isDeviceRecoverForm() {
+    return 'Recover' === document.getElementsByTagName('title')[0].innerText.split(' ')[0];
```

Comment:
wouldn't `return document.getElementsByTagName('title')[0].innerText.indexOf('Recover')  > -1` be easier to read?